src/index.html

```diff
@@ -11,7 +11,133 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
+    <form class="form" method="post">
+      <fieldset class="fieldset">
+        <legend>Personal information</legend>
+        <div>
+          <label>
+            Surname: 
+            <input name="surname" type="text" 
```

Comment:
<img width="1400" alt="Screenshot 2023-01-12 at 17 03 48" src="https://user-images.githubusercontent.com/53228259/212101933-b4f0b98d-08b9-4cc3-abf6-133d06426533.png">

general: fix it